spec.html

```diff
@@ -24894,7 +24770,7 @@ <h1>Script Records</h1>
     <emu-clause id="sec-parse-script" type="abstract operation">
       <h1>
         ParseScript (
-          _sourceText_: unknown,
+          _sourceText_: ECMAScript source text,
```

Comment:
There's an existing use of "an ECMAScript source text" in Note 1 under [The Syntactic Grammar](https://tc39.es/ecma262/multipage/notational-conventions.html#sec-syntactic-grammar). If you're changing this to omit "an", you should change that one too, to match.